docs/framework/xaml-services/x-property-directive.md

```diff
@@ -8,7 +8,7 @@ Declares a XAML property in markup.
   
 ## XAML Object Element Usage  
   
-```  
+```xaml  
 <object x:Class="className">  
   <x:Members>  
     <x:Property Name="propertyName" Type="propertyType/>  
```

Comment:
Can you try to comment the value additional properties on line 15 like this?

<!-- additionalProperties -->

The colorization looks a bit funky now.

![image](https://user-images.githubusercontent.com/12971179/64989366-3a7b0680-d882-11e9-9db5-c626721dba7f.png)